internal/importers/base/importer_test.go

```diff
@@ -146,11 +146,10 @@ func TestBaseImporter_ImportProjects(t *testing.T) {
 		importertest.TestCase
 		projects []ImportedPackage
 	}{
-		"tag constraints are ignored": {
+		"tag constraints are skipped": {
```

Comment:
For the tests, once you add back the lock, you'll need to put back `WantRevision` and `WantVersion` in the test cases.